features/google/src/views/overview/overview.jsx

```diff
@@ -40,18 +48,29 @@ const Overview = () => {
         );
     }
 
-    if (!account || files === null) {
-        return (
-            <LoadingOverlay
-                loadingGif="./images/loading.gif"
-                message={i18n.t("common:loading")}
-            />
-        );
-    }
+    const bubbleData = account.dataGroups.map(({ title, count }) => ({
+        title,
+        value: count,
+    }));
+
+    bubbleData.sort(function (a, b) {
+        return b.value - a.value;
+    });
 
     return (
         <Screen className="overview" layout="poly-standard-layout">
             {files && files?.[0] && <p>Imported File: {files[0].name}</p>}
+            <PolyChart
+                type="bubble-cluster"
+                data={bubbleData}
+                width={BUBBLE_VIZ_WIDTH}
+                height={BUBBLE_VIZ_HEIGHT}
+                bubbleColor={BUBBLE_LIGHT_COLOR}
+                textColor={BUBBLE_DARK_COLOR}
```

Comment:
If we want this **overview** screen to look the same as the one in the facebookImporter, the `textColor` should be light (or perhaps we don't even need to add it here).

Reply:
Na you're right! Non would default to white != light :D